virtualization-and-containers/material/vagrant_slides.md

```diff
@@ -0,0 +1,197 @@
+---
+type: slide
+slideOptions:
+  transition: slide
+  width: 1400
+  height: 900
+  margin: 0.1
+---
+
+<style>
+  .reveal strong {
+  font-weight: bold;
+    color: orange;
+  }
+  .reveal p {
+    text-align: left;
+  }
+  .reveal section h1 {
+    color: orange;
+  }
+  .reveal section h2 {
+    color: orange;
+  }
+</style>
+
+# Vagrant
+
+---
+
+## Introduction
+
+- Remember:
+  > If you can't git diff a file format, it's broken.
+- Infrastructure as code
+  > Vagrant is a tool for building and managing virtual machine environments in a single workflow.
+  [https://www.vagrantup.com/intro](https://www.vagrantup.com/intro)
+- Initial release 2010 by Mitchell Hashimoto, now [HashiCorp](https://www.hashicorp.com/)
+- HashiCorp develops open-source DevOps/Cloud tools like [Vagrant](https://www.vagrantup.com/), [TerraForm](https://www.terraform.io/)...
```

Comment:
```suggestion
- Remember:

> If you can't git diff a file format, it's broken.
  
- Infrastructure as code ([https://www.vagrantup.com/intro](https://www.vagrantup.com/intro))

> Vagrant is a tool for building and managing virtual machine environments in a single workflow.

- Initial release 2010 by Mitchell Hashimoto, now [HashiCorp](https://www.hashicorp.com/)
- HashiCorp develops open-source DevOps/Cloud tools like [Vagrant](https://www.vagrantup.com/), [TerraForm](https://www.terraform.io/)...
```

I think md needs these empty lines

Reply:
For code blocks it seems that Markdown wants these lines. For citations I did not get any complaints from `mdl` yet. The slides work on hedgedoc as well. Should we still change it?